http-api/src/lib.rs

```diff
@@ -232,6 +245,34 @@ pub async fn run(options: Options) -> anyhow::Result<()> {
     // Cleanup sessions
     tokio::spawn(cleanup_sessions_job(ctx.clone(), CLEANUP_SESSIONS_INTERVAL));
 
+    let app = Router::new()
+        .route("/", get(root_handler))
+        .layer(TraceLayer::new_for_http())
+        .layer(Extension(peer_id))
+        .layer(
+            CorsLayer::new()
+                .allow_origin(cors::Any)
+                .allow_methods([Method::GET, Method::POST, Method::PUT])
+                .allow_headers([CONTENT_TYPE, AUTHORIZATION]),
+        );
```

Comment:
`axum` seems to be lacking a [`.recover(...)`](https://github.com/radicle-dev/radicle-client-services/blob/master/http-api/src/lib.rs#L256). I probably have to break it down and move each response to its own respective handler. Any thoughts on that?

p.s. I've asked twice on axum discord, no answer yet. even though I see a few other companies moving (from warp and others) to axum, I'm a bit not sure atm.

Reply:
This is fixed in https://github.com/radicle-dev/radicle-client-services/pull/167/commits/3cbfd3bae0e70aa033f048a72854cf998a4da0f1. I'm also moving implementations into their respective file since lib.rs is bloated.

Implementation for each route should also go into their own respective file. Comment?

Reply:
Not sure about creating so many files, any reason to not keep them as-is in the current file?

Reply:
Will be 1 file per route e.g. `/sessions/*` will be inside `src/v1/sessions.rs`, much easier to navigate and find relevant fns. I can show you in the next PR as I've prepared it.